<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutPenSettings">
        <item>
         <layout class="QVBoxLayout" name="verticalLayoutPenLabels">
          <item>
           <widget class="QLabel" name="labelPenWidth">
            <property name="text">
             <string>Pen Width</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelPenStyle">
            <property name="text">
             <string>Pen Style</string>
            </property>
            <property name="buddy">
             <cstring>comboBoxPenStyle</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelPenCap">
            <property name="text">
             <string>Pen Cap</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelPenJoin">
            <property name="text">
             <string>Pen Join</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelPenColor">
            <property name="text">
             <string>Pen Color</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelBrushStyle">
            <property name="text">
             <string>Brush Style</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelBrushColor1">
            <property name="text">
             <string>Brush Color 1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelBrushColor2">
            <property name="text">
             <string>Brush Color 2</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayoutPenSettings">
          <item>
           <widget class="QSlider" name="horizontalSliderPenWidth">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="value">
             <number>2</number>
            </property>
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TickPosition::TicksAbove</enum>
            </property>
            <property name="tickInterval">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxPenStyle"/>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxPenCap"/>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxPenJoin"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonPenColor">
            <property name="text">
             <string>Pen Color</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxBrushStyle"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonBrushColor1">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Brush Color 1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonBrushColor2">
            <property name="text">
             <string>Brush Color 2</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutTextSettings">
          <item>
           <layout class="QVBoxLayout" name="verticalLayoutTextLabels">
            <item>
             <widget class="QLabel" name="labelText">
              <property name="text">
               <string>Text</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelFont">
              <property name="text">
               <string>Font</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelFontSize">
              <property name="text">
               <string>Font Size</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelColor">
              <property name="text">
               <string>Color</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayoutTextSettings">
            <item>
             <widget class="QLineEdit" name="lineEditText">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Text</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFontComboBox" name="fontComboBoxFont"/>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxFontSize">
              <property name="value">
               <number>16</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonTextColor">
              <property name="text">
               <string>Text Color</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="sizeConstraint">
           <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButtonCropSelect">
            <property name="text">
             <string>Select Crop Area</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonCrop">
            <property name="text">
             <string>Crop</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearButton">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>999</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAdd_Rectangle"/>
    <addaction name="actionAdd_Ellipse"/>
    <addaction name="actionAdd_Triangle"/>
    <addaction name="actionAdd_Text"/>
    <addaction name="separator"/>
    <addaction name="actionClear"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAdd_Rectangle">
   <property name="text">
    <string>Add Rectangle</string>
   </property>
  </action>
  <action name="actionAdd_Ellipse">
   <property name="text">
    <string>Add Ellipse</string>
   </property>
  </action>
  <action name="actionAdd_Triangle">
   <property name="text">
    <string>Add Triangle</string>
   </property>
  </action>
  <action name="actionAdd_Text">
   <property name="text">
    <string>Add Text</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
